wgpu-core/src/command/render.rs

```diff
@@ -568,7 +568,7 @@ impl<'a, A: HalApi> RenderPassInfo<'a, A> {
         if channel.load_op == LoadOp::Load {
             pending_discard_init_fixups.extend(texture_memory_actions.register_init_action(
                 &TextureInitTrackerAction {
-                    id: view.parent_id.value.0,
+                    id: view.parent_id.clone(),
```

Comment:
So we are now copying the refcount in here. Is that a perf tradeoff you mentioned?

Reply:
that is sure one issue, but I was mostly thinking about how setting up barriers no longer collapses (before we had less `cmd_buf_raw.transition_textures` calls) and how we resolve the texture from TextureId now in several places. Both is most prominent in `transfer.rs`